website_media_size/static/src/js/website_media_size.js

```diff
@@ -2,40 +2,32 @@
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
 
 odoo.define('website_media_size', function(require) {
+"use strict";
+
     var MediaDialog = require('web_editor.widget').MediaDialog;
     var weContext = require("web_editor.context");
+    var Widget = require('web.Widget');
 
     MediaDialog.include({
         xmlDependencies: MediaDialog.prototype.xmlDependencies.concat(
             ['/website_media_size/static/src/xml/website_media_size.xml']
         ),
         start: function() {
             var res = this._super.apply(this, arguments);
-            this.include_file_size(this.imageDialog);
-            this.include_file_size(this.documentDialog);
             this.search();
             return res;
-        },
-        include_file_size: function(obj) {
-            var self = this;
-            // ImageDialog of the web_editor.widget module is not available
-            obj.fetch_existing = function(needle) {
-                var domain = [['res_model', '=', 'ir.ui.view']].concat(obj.domain);
-                if (needle && needle.length) {
-                    domain.push('|', ['datas_fname', 'ilike', needle], ['name', 'ilike', needle]);
-                }
-                return self._rpc({
-                    model: 'ir.attachment',
-                    method: 'search_read',
-                    args: [],
-                    kwargs: {
-                        domain: domain,
-                        fields: ['name', 'mimetype', 'checksum', 'url', 'type', 'file_size'],
-                        order: [{name: 'id', asc: false}],
-                        context: weContext.get(),
-                    }
-                }).then(obj.proxy('fetched_existing'));
-            };
+        }
+    });
+
+    // ImageDialog of the web_editor.widget module is not available
```

Comment:
I took the freedom to open https://github.com/odoo/odoo/pull/23180, I'm pretty sure our friend @ged-odoo will merge that soon. :wink:

Reply:
Thanks!!